contrib/redis/patches/0005-fix-nodes-conf-ub.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/cluster.c b/src/cluster.c
```

Comment:
you should also send this upstream if you want

Reply:
the cfi fixes might be work sending upstream, too